### ISSUE TITLE
Travis - Test Bitcoin anchoring

### ISSUE THREAD

**Issue by lautarodragan** [poetapp/node#25]
Configure Travis to run it and write an integration test that:
[ ]  POST /work 
[ ]  GET /work/:id 
[ ]  Expect work.txid to be defined, etc.
[ ] Verify work

### Optional
The bitcoind docker container will need to generate blocks periodically for broadcasted transactions to be confirmed. (Make another ticket for this?)

### Blockers
This task depends on having a bitcoind -regtest docker image (see #21) and adapting the Node to use Bitcoin's RPC instead of Insight (see #24)

Needs #521

**Comment by geoffturk** [comment]
needs poetapp/node#21

**Comment by warrenv** [comment]
See update on functional test progress here: https://github.com/poetapp/random/issues/137#issuecomment-415646787

Once that is merged, we canl work on the travis integration.
